### DISCUSSION TITLE
Bluetooth not connecting to Sideband

### DISCUSSION THREAD

**Discussion by LuxStatera** [markqvist/Reticulum#335] [Help & Questions]
I connected my RNode with USB to my Android and opened Sideband. Connectivity is set to RNode and Hardware is set to allow Bluetooth. I paired the node and shutdown Sideband. Unplugged the node and then powered it with a battery. I then open Sideband but it shows the RNode interface is down and it is not connecting with bluetooth. I am using a LILYGO LoRa32

**Comment by markqvist** [comment]
Does it work ok over the cabled connection? Did you also enable bluetooth on the RNode (ie., is the bluetooth symbol enabled on the display)?

**Comment by markqvist** [comment]
Transferring to discussions, as this is likely not an issue with Reticulum.

**Comment by faragher** [comment]
In trying to test this, I have been unable to pair the RNode with my phone. It keeps complaining the PIN or passcode is wrong. I'm pretty sure I got it to connect before, and I have no way to connect the node directly to my phone. I'm going to try to find the RNode it was previously paired with, but this one, firmware 1.61, doesn't seem to want to pair.

Edit: Never mind, RNodeconf pairing mode. Testing.

**Comment by faragher** [comment]
I am having a similar problem. Got the RNode paired, enabled RNode using _Connect using Bluetooth_ in Hardware/RNode menu and _Connect via RNode_ in Connectivity menu. No other interfaces are enabled. Bluetooth indicator is ready (B) not active (-B-) and waterfall is blank. Cable indicator is an S shape, but is on voltage only USB cable. Identical results on use of battery attached to LilyGo JST connector.

Attempts to announce or check for new messages occur without any error, but obviously return no data. Would it be helpful to set up a second RNode and try live transmissions?

Edit: I realized I had another radio up and running, so I just did the test. See below.

**Reply by faragher** [reply]
Sideband configuration is very sticky. Despite closing the app via swiping it closed, it continued to transmit over disabled internet route. Force stopped app, now shows connection, waterfall, and Sideband logo on external framebuffer. System broadcasts and receives properly.

Lux, please check the configuration as above and force-quit (long press on icon, app info, force quit) then restart the app.

Mark, I suggest a way for rnsd or Sideband equivalent to close all interfaces and reload configuration via API or otherwise not require a hard restart.

**Reply by cobraPA** [reply]
Remember that Reticulum/rnsd runs the networking layer and depending on config it will start as a service in the background.  Sideband would then use it, but restarting Sideband will not reload the network config yet.  I think you are using Linux?  It is probably a service then (rnsd) and you would need to restart the service to get the network changes to stick.  (I think I got all the right... welcome for corrections or platform details.  The Android version works a little differently...)

**Reply by faragher** [reply]
Correct, but this is the Android Sideband, which I believe is an entirely internal Reticulum stack.

**Reply by markqvist** [reply]
To properly close/restart Sideband and tge RNS service, you need to use the "Shutdown" option in the menu, which signals the background service to close. Unless you do that, configuration will *not* be reloaded. Swiping app away is not enough, it only removes the foreground activity. This is on purpose.
